garak/buffs/base.py

```diff
@@ -72,6 +73,12 @@ def transform(
         """attempt copying is handled elsewhere. isn't that nice"""
         yield self._derive_new_attempt(attempt)
 
+    def untransform(
+        self, attempt: garak.attempt.Attempt
+    ) -> garak.attempt.Attempt:
+        """Only necessary if buff requires a post buff hook"""
+        pass
```

Comment:
seeing as post_buff_hook gates calling untransform, how do you feel about raising a `NotImplementedError` here, or returning  `attempt`?